plenum/server/replica.py

```diff
@@ -2410,6 +2419,17 @@ def caught_up_till_3pc(self, last_caught_up_3PC):
         self._remove_ordered_from_queue(last_caught_up_3PC)
         self.update_watermark_from_3pc()
 
+    def catchup_clear_for_backup(self):
```

Comment:
We should perform the following actions only if the replica is not primary.

(In case of performing catch-up in scope of view change, the primary status will be undefined, but before a new view we may clear all the replicas. So we should perform cleaning in any case when the replica is not being primary.)

Reply:
done